subprojects/testing-native/testing-native.gradle

```diff
@@ -19,6 +19,7 @@ dependencies {
     compile project(':platformNative')
     compile project(':languageNative')
     compile project(':testingBase')
+    compile project(':testingJvm')
```

Comment:
The shared stuff should live in `testingBase`, this project should have nothing to do with the JVM projects. It's ok for now, but after we detangle `XcTest` from `Test` this dependency should go away.